### PR TITLE
[8]Add Page Navigation to Pages

### PR DESCRIPTION
**Pull request by Cynthia-Berry**
## Task: `[8]Add Page Navigation to Pages to Pages`
- `I ensured pages are properly linked`
home page
contact

### CODE REVIEW

File: submissions/cynthiaNwakaeme/HTML/index.html [hunk 1]
```diff
@@ -13,18 +13,16 @@
```
Comment (ToluAkin) [inline submissions/cynthiaNwakaeme/HTML/index.html:13]: @Cynthia-Berry Is this your product page?

**Comment by Cynthia-Berry** [comment]
@ToluAkin Yes ma.. index.html is my product page

File: submissions/cynthiaNwakaeme/HTML/index.html (hunk 1 above)
Comment (ToluAkin) [reply submissions/cynthiaNwakaeme/HTML/index.html:13]: @Cynthia-Berry You did not link your signup form?

Comment (Cynthia-Berry) [reply submissions/cynthiaNwakaeme/HTML/index.html:13]: Hello @ToluAkin .
I have effected the changes as requesterd.
Kindly approve my PR.
Thank you so much

**Pull request merged**